.ci/magician/cmd/templates/vcr/record_replay.tmpl

```diff
@@ -1,11 +1,19 @@
 {{- if gt (len .RecordingResult.PassedTests) 0 -}}
 {{color "green" "Tests passed during RECORDING mode:"}}
-{{range .RecordingResult.PassedTests}}`{{.}}`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-{{$.PRNumber}}/artifacts/{{$.BuildID}}/recording/{{.}}.log)]
+{{range .RecordingResult.PassedTests -}}
+`{{.}}` {{/* remove trailing whitespace */ -}}
```

Comment:
Seems like this is very intentional, but I'm curious what it is doing / why it is needed

Reply:
It's more needed for L13-15 and L32-34, but I did it here as well for consistency.

The idea is to make the template more readable by limiting the line length. The comment removes the newlines I inserted from the output so that it remains unchanged apart from adding a space between the test name and the link to the debug log.

The comment on L6 is just to ensure there's always a newline between each test entry. Leaving off the `-` on L3 would have the same effect.

Reply:
A more ideal solution would be to use a subtemplate or a template function to format the URLs, but I'll save that for a later PR when we do a general refactor of these templates.